packages/web/components/Form/MaskedInputField.js

```diff
@@ -91,7 +92,7 @@ const MaskedInputField = ({
 				/>
 				{help && <Help id={name} label={label} HelpComponent={help} />}
 			</Row>
-			{errors && Object.keys(errors).length ? (
+			{Object.keys(errors)?.length ? (
```

Comment:
hey man, there's a problem in that block

open your devtools and type:

```
const errors = undefined
Object.keys(errors)?.length
```

BOOM 💣 

`Cannot convert undefined or null to object`

That's why we need to check it

Reply:
**Wow!**  🤯 🧠 

You rock, bro! 😎 

What do you think about doing this? 👇
```suggestion
			{Object.keys(errors || {})?.length ? (
```

Okay, I'm kidding, you know...

Reply:
hahahaha sick!